sdk/eventgrid/Azure.Messaging.EventGrid/api/Azure.Messaging.EventGrid.netstandard2.0.cs

```diff
@@ -1341,3 +1341,12 @@ public partial class WebSlotSwapWithPreviewStartedEventData
         public string Verb { get { throw null; } }
     }
 }
+namespace Microsoft.Extensions.Azure
```

Comment:
"Add a blurb" you say. Sneaky :)

Do we want to try and add tests for these?

Reply:
Created https://github.com/Azure/azure-sdk-for-net/issues/17158